lol.janjic.Splitcat.yml

```diff
@@ -0,0 +1,48 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+---
+app-id: lol.janjic.Splitcat
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: splitcat
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+modules:
+  - name: Splitcat
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - |
+        # No spaces in project name.
+        projectName=splitcat
+        projectId=lol.janjic.Splitcat
+        executableName=splitcat
+
+        # ------------------------------- Build Flatpak ----------------------------- #
+
+        # Extract portable Flutter build.
+        mkdir -p $projectName
+        tar -xf $projectName-linux-x64.tar.gz -C $projectName
+
+        mv splitcat /app/
+        chmod +x /app/splitcat/splitcat
+        mkdir -p /app/bin
+        ln -s /app/splitcat/splitcat /app/bin/splitcat
+
+        wget https://raw.githubusercontent.com/vogonwann/splitcat/d8f95eff7dbfaf283dbfaf93b64ebbd2302d7847/linux/flutter/assets/icons/256x256/icon.png -o $iconDir/$projectId.png
+
+        install -Dm644 packaging/linux/$FLATPAK_ID.desktop -t $FLATPAK_DEST/share/applications/
+        install -Dm644 packaging/linux/$FLATPAK_ID.appdata.xml -t $FLATPAK_DEST/share/metainfo
```

Comment:
as it is right now the source files don't exist, so this will need to be adjusted.

Reply:
I'll add it

Reply:
done

Reply:
v0.1.5

Reply:
files are in the root of the archive

Reply:
also remove desktop file, metainfo and icon from here.

Reply:
will do

Reply:
removed

Reply:
> icon is not in the archive.

I'll add it

Reply:
added